### ISSUE TITLE
Updating RemoteProxy with Tooling API fails with deserialization error

### ISSUE THREAD

**Issue by mrichmon12** [forcedotcom/cli#72]
### Summary

Receive an error when trying to update RemoteProxy object through force:data:record:update

### Steps To Reproduce:

1.  Attempt to update a RemoteProxy through the force:data:record:update
```shell
sfdx force:data:record:update \
    --targetusername DEV \
    --sobjecttype RemoteProxy \
    --sobjectid 0rpf2000000EJMtAAO \
    --usetoolingapi \
    -v "Metadata='{\"disableProtocolSecurity\": false,\"isActive\": true,\"url\": \"https://www.example.com\",\"urls\": null,\"description\": null}'" 
```

### Expected result

RemoteProxy is updated in the target org.

### Actual result

ERROR:  Cannot deserialize instance of complexvalue from null value null or request may be missing a required field.

### Additional information

The Metadata format used above matches the format that is output from a query of the RemoteProxy object, and it does work fine via the Tooling API directly outside of the field-value-pair formatting change specified in the CLI reference.

[Salesforce DX Trailblazer Community post](https://success.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F93A000000HTp1&fId=0D53A00004ETCR4&s1oid=00D300000000iTz&emkind=chatterCommentNotification&s1nid=0DB30000000072L&emtm=1554741778026&s1uid=0053A00000CLwlD&fromEmail=1&s1ext=0)

**SFDX CLI Version**
sfdx-cli/7.2.0-179c9078fc linux-x64 node-v10.15.3

**SFDX plugin Version**
@oclif/plugin-commands 1.2.2 (core)
@oclif/plugin-help 2.1.6 (core)
@oclif/plugin-not-found 1.2.2 (core)
@oclif/plugin-plugins 1.7.8 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.2 (core)
analytics 1.1.2 (core)
generator 1.1.0 (core)
salesforcedx 45.9.1 (core)
├─ force-language-services 45.9.0 (core)
└─ salesforce-alm 45.12.0 (core)

**OS and version**:
Windows 10 Enterprise

**Comment by danielepiccone** [comment]
I am getting the same issue when trying to toggle a flow with

```
sfdx force:data:record:update -u sandbox -t -s FlowDefinition -i 300b000000008ohAAA -v "Metadata='{\"activeVersionNumber\": 2, \"description\": null, \"masterLabel\": null, \"urls\": null}'"
```

I figured out this call ends up not sending the correct payload.

Apparently the JSON is sent as a plain string instead of being properly parsed.

I managed to get it to work by patching `/node_modules/salesforce-alm/dist/lib/data/dataRecordUpdateCommand.js` but I cannot find the source code for this repository online to open a PR.

**SFDX CLI Version**
sfdx-cli/7.36.0-b1c4e9c7c8 darwin-x64 node-v10.15.3

**SFDX plugin Version**
no plugins

**OS and version:**
Darwin / OSX 10.14.6

**Comment by mshanemc** [comment]
@danielepiccone that code is OSS and lives here: https://github.com/salesforcecli/plugin-data if you would like to fix it.

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-10290524](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000WjD9xYAF/view)

**Comment by iowillhoit** [comment]
This issue has been fixed and included in today's stable release `7.138.1`. [Update](https://developer.salesforce.com/docs/atlas.en-us.234.0.sfdx_setup.meta/sfdx_setup/sfdx_setup_update_cli.htm) to the latest `sfdx` to get these changes. Release notes can be viewed [here](https://github.com/forcedotcom/cli/tree/main/releasenotes/sfdx#71381-feb-17-2022-stable). Thank you!
